core/src/main/java/hudson/console/PlainTextConsoleOutputStream.java

```diff
@@ -64,7 +70,11 @@ protected void eol(byte[] in, int sz) throws IOException {
             int rest = sz - next;
             ByteArrayInputStream b = new ByteArrayInputStream(in, next, rest);
 
-            ConsoleNote.skip(new DataInputStream(b));
+            try {
+                ConsoleNote.skip(new DataInputStream(b));
+            } catch (IOException x) {
+                LOGGER.log(Level.FINE, "Failed to skip annotation from \"" + StringEscapeUtils.escapeJava(new String(in, next, rest, Charset.defaultCharset())) + "\"", x);
```

Comment:
Maybe better to just log the raw byte array for debugging purposes since we don't know what charset was used at this level.

Reply:
Well, normally ASCII for the note itself, but the `escapeJava` will take care of any escapes. Note that Pipeline is always UTF-8, and Java will always be UTF-8 in a future version, so I am not too concerned.

BTW this is simply copied from https://github.com/jenkinsci/jenkins/blob/83085bac0081f7366abf9c74616b1b1a9ad62bb3/core/src/main/java/hudson/console/ConsoleAnnotationOutputStream.java#L125